agimus_demos_common/agimus_demos_common/static_transform_publisher_node.py

```diff
@@ -0,0 +1,37 @@
+from launch_ros.actions import Node
+import typing as T
+
+
+def static_transform_publisher_node(
+    frame_id: str,
+    child_frame_id: str,
+    xyz: T.Optional[T.Tuple[float, float, float]] = None,
+    rot_xyzw: T.Optional[T.Tuple[float, float, float, float]] = None,
+    rot_rpy: T.Optional[T.Tuple[float, float, float]] = None,
+    node_kwargs={},
+) -> Node:
+    assert rot_rpy is None or rot_xyzw is None
```

Comment:
I am not sure what you are trying to test here because if they are both None then what do you do?

Reply:
I'm checking here that the user does not pass both a quaternion and a RPY vector.

Reply:
I have updated the code.